ada-project-docs/wave_03.md

```diff
@@ -1,18 +1,19 @@
-# OPTIONAL Wave 3: Enhancements & Deployment
+# Wave 3: Enhancements & Deployment
 
-These really are **optional** - if you've gotten here and you have time left, that means you're moving speedy fast!
-
-## Enhancements
-
-### Query Parameters
-Any endpoint that returns a list should accept 3 _optional_ query parameters:
+## Query Parameters
+The following 3 _optional_ query parameters:
 
 | Name   | Value   | Description
 |--------|---------|------------
 | `sort` | string  | Sort objects by this field, in ascending order
 | `n`    | integer | Number of responses to return per page
 | `p`    | integer | Page of responses to return
```

Comment:
Would it be possible to update these query parameter names to something like `count` and `page`? I know there are APIs that absolutely use single letter query parameter names, but I think it would be nice to model using short but descriptive names over single letters.